redbot/cogs/admin/announcer.py

```diff
@@ -77,5 +77,5 @@ def cancel(self):
         )
         if failed:
             msg += humanize_list(tuple(map(inline, failed)))
```

Comment:
```
		if failed:
			msg = (
        	    _("I could not announce to the following server: ")
        	    if len(failed) == 1
       		    else _("I could not announce to the following servers: ")
       		)
			msg += humanize_list(tuple(map(inline, failed)))
			await self.ctx.bot.send_to_owners(msg)
```
Would be better here There is no purpose to assing and validate that string here  unless there's been failed attempts